### DISCUSSION TITLE
Error when loading the user list

### DISCUSSION THREAD

**Discussion by DennisNissenUX** [Budibase/budibase#7083] [Help]
Hey guys,

I wanted to add a new user to our space and I found this error and can't find the error on my dev side.
I don't want to rule out the possibility that it is related to our own hosting. I still wanted to ask if anyone has encountered a similar problem and if anyone might have a solution suggestion.

My problems are:
- The list of users can no longer be loaded
- I can not add a user

I am grateful for any advice. If any other information from my setup is relevant, please feel free to let me know so I can look it up.

<img width="1428" alt="Bildschirmfoto 2022-08-04 um 13 13 31" src="https://user-images.githubusercontent.com/105709029/182833622-8f68b5a1-d447-419a-9237-72025058011d.png">

**Comment by R2bEEaton** [comment]
Hello! It looks like your installation is out of date - there was a recent User Management overhaul. I'd say the first thing you should try is updating your self-hosted installation.

**Comment by DennisNissenUX** [comment]
Okay, thanks for the response! I will try and see if I can get it to work this way 👍

**Reply by R2bEEaton** [reply]
Hey @DennisNissenUX, any update on your situation?

(Also, for future reference, you can reply to a GitHub comment chain by using the "Write a reply" text box directly below.)

**Reply by DennisNissenUX** [reply]
Hey, we couldn't really find the cause of the problem yet. We updated the installation. We are currently on 1.1.28

it seems that something with the search is wrong. Any idea what we can do? 

![Bildschirmfoto 2022-08-08 um 17 49 55](https://user-images.githubusercontent.com/105709029/183459386-0ec4f9b9-88be-4b99-b1fc-d29d57a39203.png)

**Reply by R2bEEaton** [reply]
Hmm, that's odd. My search is coming through fine. Can you post some more setup information?

**Reply by DennisNissenUX** [reply]
Here are some more screenshots of the errors. Could it be that the "search" frame is missing or something? What more information do you need?

![Bildschirmfoto 2022-08-08 um 22 09 08](https://user-images.githubusercontent.com/105709029/183506540-0dff5033-9763-41ab-9ffd-2c9a0b8f2d7d.png)

![Bildschirmfoto 2022-08-08 um 22 14 52](https://user-images.githubusercontent.com/105709029/183506531-e3ca6f14-6624-4271-b538-e23156b83410.png)

**Reply by R2bEEaton** [reply]
Information like how you are hosting (Docker, Single Docker, Kubernetes, etc.).

It might be necessary to export your app(s) and do a fresh install.

**Reply by DennisNissenUX** [reply]
We are hosting budibase on kubernetes and have installed it with Helm Charts.
I think that maybe a fresh install might do the trick. Unfortunately, I think we will lose our user data that way, so that we will need to add all users again.
Do you have another idea maybe? Or else I think we will need the fresh start.

**Reply by shogunpurple** [reply]
Hi @DennisNissenUX - this sounds like an issue we’ve seen in the past with kubernetes setups and the version of CouchDB being used. 

I still was not able to recreate this, but if you downgrade the version of your couchDB chart, it seems to fix the issue.

More information here: https://[github.com/Budibase/budibase/issues/6168#issuecomment-1154026447](https://github.com/Budibase/budibase/issues/6168#issuecomment-1154026447)

You can update your couchDB values in our values.yaml to the following:

```
# The CouchDB image
image:
 repository: couchdb
 tag: 3.1.0
 pullPolicy: IfNotPresent

```

Hope this helps!

**Reply by DennisNissenUX** [reply]
Thank you! I have submitted the infos to our admin, who wants to try it.
I will give feedback as soon as I know if it worked or not

**Reply by DennisNissenUX** [reply]
Hey guys, quick update.

your comments were extremely helpful and described the problem in detail. We still had to set up the hosting again, because the described solution had caused new problems. Now it is working again! 

Thanks again guys
